src/Detector/Python/PythonDetector.cs

```diff
@@ -66,6 +66,11 @@ private bool IsPythonApp(ISourceRepo sourceRepo)
             }
 
             var searchSubDirectories = !_options.DisableRecursiveLookUp;
+            if (!searchSubDirectories)
+            {
+                _logger.LogDebug("Skipping search for files in sub-directories as it has been disabled.");
```

Comment:
add same logging for hugo, node, and php?

Reply:
They currently do not do a recursive look up so did not need to add it there